app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    source_file = None
+    try:
+        _, source_file, destination_path = command.split()
```

Comment:
`Make sure you check that the mv input command is correct.`
this requirement is not satisifed